rboot-ota/rboot-ota.c

```diff
@@ -51,7 +53,7 @@ bool ICACHE_FLASH_ATTR rboot_set_config(rboot_config *conf) {
 	
 	buffer = (uint8*)os_malloc(SECTOR_SIZE);
 	if (!buffer) {
-		uart0_send("No ram!\r\n");
+		os_sprintf("No ram!\n");
```

Comment:
i'm happy to switch to os_sprintf, but probably want to do this everywhere, not just this one file
please leave \r as \r\n seems to be the official sdk way to do things